src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -450,6 +450,10 @@ export class HuiDialogEditCard extends LitElement
         }
         .element-preview {
           position: relative;
+          height: max-content;
```

Comment:
This is set to make sure the height of the preview does not grow with the left side. Example: Lots of entities in the entities card